### ISSUE TITLE
Decode array of URLs

### ISSUE THREAD

**Issue by ishaq** [hkellaway/Gloss#81]
right now this would fail

```swift
let imageURLs:[NSURL] = ("imageURLs" <~~ json)!
```

we have to do this instead:

```swift
// Get an array of strings first
let imageURLStrings:[String] = ("imageURLs" <~~ json)!
// now convert it to an array of URLs manually
var imageURLs:[NSURL] = []
for var i = 0; i < imageURLStrings.count; i++ {
    let currentURL = NSURL(string: imageURLStrings[i])!
    imageURLs.append(currentURL)
}
```

I'll try to figure out how to do it in Gloss and submit a PR. Or if it's a quick fix, I'll let @hkellaway take care of it.

**Comment by hkellaway** [comment]
good call @ishaq - i will fix this

**Comment by hkellaway** [comment]
not available if you point your pod/cartfile to `develop` - will be available in the next, `0.6.1`, release

**Comment by ishaq** [comment]
great :-)
